scripts/joinmarket-qt.py

```diff
@@ -527,7 +530,7 @@ def initUI(self):
         buttons.addWidget(self.startButton)
         buttons.addWidget(self.abortButton)
         self.abortButton.clicked.connect(self.abortTransactions)
-        innerTopLayout.addLayout(buttons, len(self.widgets) + 1, 0, 1, 2)
```

Comment:
It's a shame to lose flexibility by hardcoding 5. But a really minor point so OK.